packages/universal-swap/src/helper.ts

```
@@ -328,20 +333,46 @@ export class UniversalSwapHelper {
+    injAddress?: string;
+    evmInfo?: {
+      [key: string]: string;
     };
+  }) => {
+    const addressFollowCoinType = { address60: injAddress, address118: oraiAddress };
+    let cosmosAddress = cosmosChains.reduce(
+      (acc, cur) => {
+        if (!injAddress && cur?.bip44?.coinType === 60) return acc;
+        if (!acc?.[cur.chainId]) {
+          return {
+            ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)